Extension/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.5.0-insiders: June 14, 2021
+### Enhancements
+* Add "Symbol Options" for CppVsdbg to configure symbol settings [PR #7680](https://github.com/microsoft/vscode-cpptools/pull/7680)
+* Update CppVsdbg to use newer CppEE and msdia. 
+
+### Bug Fixes
+* Fix Find All References reporting certain references in headers as inactive. [#7609](https://github.com/microsoft/vscode-cpptools/issues/7609)
+* Fix IntelliSense process crash and tag parser failure with columns > 65535. [#7621](https://github.com/microsoft/vscode-cpptools/issues/7621)
+* Fix incorrect translations.
```

Comment:
nit: change to "Fix incorrect localization translations." so it's less ambiguous.

Note: already updated this in internal copy on OneNote.

Reply:
You mean translation could be interpreted as a non-localization translation, i.e. like geometric translation?

Reply:
or any kind translation, like I was thinking translation unit initially. So suggested to add "localization" to clarify.

Reply:
They were French translations, so you could say "French translations" to remove any ambiguity.

Reply:
I think some were portugese too in another change.